src/Fixer/ClassNotation/NoUnneededFinalMethodFixer.php

```diff
@@ -61,34 +67,31 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
 
             $classOpen = $tokens->getNextTokenOfKind($index, ['{']);
             $prevToken = $tokens[$tokens->getPrevMeaningfulToken($index)];
-            if (!$prevToken->isGivenKind(T_FINAL)) {
-                $index = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_CURLY_BRACE, $classOpen);
+            $classIsFinal = $prevToken->isGivenKind(T_FINAL);
 
-                continue;
-            }
-
-            $index = $this->fixClass($tokens, $classOpen);
+            $this->fixClass($tokens, $classOpen, $classIsFinal);
```

Comment:
why this change? like, without `$index = ...` assignment we don't do the jump, so it's slower to finish the main loop. do we need to not do that jump ?

Reply:
If we jump to the end of the class and ignore the content of methods, we cannot fix anonymous class definitions inside methods. 

Do you have a suggestion, @keradus?

Reply:
with that reasoning, keep your change :+1: